src/components/DatePickerContainer/DatePickerContainer.css

```diff
@@ -0,0 +1,11 @@
+.infoPanel {
+	padding: 10px 21px;
+	border-top: 1px solid #dce0e0;
+	color: #484848;
+}
+
+.info {
```

Comment:
Oh oops. I forgot, I got rid of it because I wasn't sure if it was needed but then I didn't get rid of the css styling. 

It was supposed to be on: https://github.com/opentraffic/analyst-ui/blob/mkong0216/calendar/src/components/DatePickerContainer/index.js#L47

Since I already removed the span, should I just remove the styling for the span. Or would you want to see how it looks with the styling?